src/consensus/find_fork_point.py

```diff
@@ -1,11 +1,12 @@
-from typing import Dict, Union
+from typing import Union
 
+from src.consensus.blockchain_interface import BlockchainInterface
 from src.consensus.sub_block_record import SubBlockRecord
 from src.types.header_block import HeaderBlock
 
 
 def find_fork_point_in_chain(
```

Comment:
Maybe write a comment saying that this method assumes that the fork point is included within the sub_blocks.